Documentation/PageTsconfig/TceForm.rst

```diff
@@ -230,10 +230,10 @@ In this example, the value will substitute the marker in a plugin FlexForm.
 ..  code-block:: typoscript
     :caption: EXT:site_package/Configuration/page.tsconfig
 
-    TCEFORM.tx_myext_table.myfield.PAGE_TSCONFIG_ID = 22
+    TCEFORM.table.myfield.PAGE_TSCONFIG_ID = 22
```

Comment:
```suggestion
    TCEFORM.tx_myextension_table.myfield.PAGE_TSCONFIG_ID = 22
```

I would like to stick with a realistic table name. All extension provided tables have to start with tx.